core/src/com/unciv/logic/trade/TradeEvaluation.kt

```diff
@@ -158,6 +162,16 @@ class TradeEvaluation {
             }
         }
     }
+    fun isSurroundedByOurCities(city: CityInfo, civInfo: CivilizationInfo): Int{
```

Comment:
By general coding convention, functions starting with "is" are expected to be Boolean. If this returns something else, change the function name accordingly.

Reply:
Going over it, I don't understand what this number is supposed to represent.

Reply:
The number means the number that is added to the function that determines the value of the city.

if the city is completely surrounded by the target traded, that the AI will pay 10* Civ Era

if the city is bordered by the target traded, that the AI will pay 2* Civ Era

Reply:
fixed